posts/2022/office-hours-appointments.md

```diff
@@ -0,0 +1,19 @@
+---
+author: Julia Kent
+date: 2022-08-10
+tags: office-hours
+---
+
+# New Office Hours Appointment System
+
+The ESDS office hours are moving to an [appointment scheduling page](office-hours/).
+
+The previous office hours system was two one-hour blocks at the same time every week via a Zoom call where software engineers sit in and await scientists to show up and receive help. This was not ideal because some weeks noone showed up (which eventually leads to lower attrition of software engineers), and other weeks many people showed up and could not all receive the individualized help they needed. Also we had many software engineers and scientists with scheduling conflicts during these two static time blocks. You may have noticed these are no longer on the ESDS calendar.
```

Comment:
```suggestion
The previous office hours system was two one-hour blocks at the same time every week via a Zoom call where software engineers sit in and await scientists to show up and receive help. This was not ideal because some weeks no one showed up (which eventually leads to lower attrition of software engineers), and other weeks many people showed up and could not all receive the individualized help they needed. Also we had many software engineers and scientists with scheduling conflicts during these two static time blocks. You may have noticed these are no longer on the ESDS calendar.
```

Typo